src/MLJDecisionTreeInterface.jl

```diff
@@ -129,13 +129,66 @@ function MMI.fit(
                              m.min_samples_split,
                              m.min_purity_increase;
                              rng=m.rng)
-    cache  = nothing
+    cache  = deepcopy(m)
 
     report = (features=features,)
 
     return (forest, classes_seen, integers_seen), cache, report
 end
 
+function MMI.update(
+    model::RandomForestClassifier,
+    verbosity::Int,
+    old_fitresult,
+    old_model,
+    Xmatrix,
+    yplain,
+    features,
+    classes,
+    )
+
+    only_iterations_have_changed = MMI.is_same_except(model, old_model, :n_trees)
+
+    if !only_iterations_have_changed
```

Comment:
It might be worth adding a `@info` statement for when the model is retrained from scratch. Something along these lines:

```verbosity > 0 && @info "Retraining model. More parameters than `:n_trees` have changed"```

Reply:
Done, with a bit different wording.

Reply:
Hang on. I only did the classifier but the new tests are still passing ?!?

Reply:
Oh, I had the wrong branch checked out locally...

Reply:
Okay, I think I got there in the end.

Reply:
This looks great. Resolved